### PR TITLE
CTCP-3765: IE013 - Amendment Journey - Against IE055 (For a Standard Movement)

### PR DESCRIPTION
**Pull request by joecefnhedog**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -145,7 +145,7 @@ microservice {
     features {
       isGuaranteeBalanceEnabled = true
       phase5Enabled {
-        departure = false
+        departure = true
```
Comment (joecefnhedog) [inline conf/application.conf:148]: Need to change this back

Comment (Reece-Carruthers) [reply conf/application.conf:148]: Yeah that'll need to be set back to false

Comment (joecefnhedog) [inline conf/application.conf:148]:
```suggestion
        departure = false
```

File: test/views/departure/testOnly/GuaranteeRejectedP5ViewSpec.scala [hunk 2]
```diff
@@ -33,6 +33,7 @@ class GuaranteeRejectedP5ViewSpec extends ViewBehaviours with Generators {
   override val prefix: String = "guarantee.rejected.message"
 
   private val guaranteeReferences: Seq[GuaranteeReference] = Gen.nonEmptyListOf(arbitrary[GuaranteeReference]).sample.value
+//  private val departureId: String                          = "departureId123"
```
Comment (joecefnhedog) [inline test/views/departure/testOnly/GuaranteeRejectedP5ViewSpec.scala:36]: Just spotted this, will delete before merge

File: test/views/departure/testOnly/GuaranteeRejectedP5ViewSpec.scala [hunk 3]
```diff
@@ -44,7 +45,7 @@ class GuaranteeRejectedP5ViewSpec extends ViewBehaviours with Generators {
 
   override def view: HtmlFormat.Appendable = injector
     .instanceOf[GuaranteeRejectedP5View]
-    .apply(defaultViewModel)(fakeRequest, messages)
+    .apply(defaultViewModel, "d")(fakeRequest, messages)
```
Comment (joecefnhedog) [inline test/views/departure/testOnly/GuaranteeRejectedP5ViewSpec.scala:48]: Just spotted this, will delete update before merge

**Pull request merged**